src/components/ContactsList/ContactsTable.tsx/useSortedContacts.tsx

```diff
@@ -0,0 +1,13 @@
+import { Contact, ContactAwaitingAcceptance } from '@/api/types/contact'
+import { useMemo } from 'react'
+
+export const useSortedContacts = (contacts: Contact[] | null) => {
```

Comment:
Our current backend doesn't support sorting this way. 
Because we have: 
```
{
  order: string
  sort: 'desc' | 'asc'
}
```
And we canno receive the list where `pendig` contacts are at the begining of the list.
So, we could leave this `useSortedContacts` cloent-side sort or modify backend behaviour.

Reply:
or we can make 2 separate requests, 
one for `pending` contacts
and one for not `pending`
which maybe isn't that bad in this case, and it won't force the backend to have special logic of handling it?

Reply:
good point. I can do that 👍even in this PR.

Reply:
As we agreed, lets keep the sorting on frontend

Reply:
According to our team meeting - for now, we're not going to change client-side sorting.